docs/quickstart/requirements.md

```diff
@@ -0,0 +1,30 @@
+---
+hide:
+    - toc
+
+title: System Requirements
+---
+
+Since the Inspector has two versions, [a desktop app and a web app](../overview.md#formats), the
+requirements for these will differ.
+
+-   The web app supports Chrome and Firefox browsers
+    ([Safari is not supported](../troubleshooting.md#browser-version-does-not-work-in-safari))
+    -   Viewport size of at least **870 x 610** pixels is recommended
+-   The desktop app requires up to around **600MB** of space on the computer
+    -   The minimum application window size is **890 x 710** pixels
+
+Both Inspector versions also require an **Appium server** to connect to, which is _not_ bundled with
+the Inspector. This server can be run either locally or remotely. If you need to install it, please
+check the [Appium Install documentation](https://appium.io/docs/en/latest/quickstart/install/)
+for details.
+
+!!! note
+
+    If using a standalone Appium server, make sure the server also has the necessary
+    [driver(s)](https://appium.io/docs/en/latest/ecosystem/drivers/) for your target platform(s)!
+
+While the Inspector is designed to work with Appium 2, it is also compatible with
```

Comment:
Not sure if we even need to mention Appium 1 except of the fact it has been EOLed a while ago

Reply:
I think it's reasonable to mention here, given that most things in the Appium ecosystem are no longer compatible with Appium 1. There will still be a point where Appium 1-specific workarounds (such as `automationName` not being guaranteed) will be dropped, and the docs updated accordingly.